GraphAlgorithms/Clique/MaxCliqueFinder.cs

```diff
@@ -47,12 +47,12 @@ public List<int> FindHeuristic()
 
         for (int i = 0; i < _graph.Size; i++)
         {
-            if (_degrees[i] >= _maxClique.Count)
+            if (_degrees[i] >= _maxClique.Count - 1)
```

Comment:
These -1 are so that we continue even if we know we wont find a larger clique (more vertices) because we could still find one with more edges. we could separate different ways of calculating size into 2 separate algorithms for minor performance improvements but I thinks its overkill.